tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/XUnit/XUnitIntegration.cs

```diff
@@ -6,144 +6,121 @@
 
 using System;
 using System.Collections.Generic;
-using System.Reflection;
-using System.Threading;
 using Datadog.Trace.Ci;
 using Datadog.Trace.Ci.Tags;
 using Datadog.Trace.Configuration;
 
-namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.XUnit
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.XUnit;
+
+internal static class XUnitIntegration
 {
-    internal static class XUnitIntegration
-    {
-        internal const string IntegrationName = nameof(Configuration.IntegrationId.XUnit);
-        internal const IntegrationId IntegrationId = Configuration.IntegrationId.XUnit;
+    internal const string IntegrationName = nameof(IntegrationId.XUnit);
+    internal const IntegrationId IntegrationId = Configuration.IntegrationId.XUnit;
 
-        internal static bool IsEnabled => CIVisibility.IsRunning && Tracer.Instance.Settings.IsIntegrationEnabled(IntegrationId);
+    internal static bool IsEnabled => CIVisibility.IsRunning && Tracer.Instance.Settings.IsIntegrationEnabled(IntegrationId);
 
-        internal static Scope? CreateScope(ref TestRunnerStruct runnerInstance, Type targetType)
+    internal static Test? CreateScope(ref TestRunnerStruct runnerInstance, Type targetType)
```

Comment:
nit: Like the other framework integrations, could you rename this to `CreateTest` and `FinishTest`?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3276/commits/c7f88265bbdc698ca8f9fd86f24c3e5df47a6b5b